modules/prebidServerBidAdapter/index.js

```diff
@@ -658,10 +693,26 @@ const OPEN_RTB_PROTOCOL = {
             bidRequest.serverResponseTimeMs = serverResponseTimeMs;
           }
 
-          const extPrebidTargeting = utils.deepAccess(bid, 'ext.prebid.targeting');
+          // Look for seatbid[].bid[].ext.prebid.bidid and place it in the bidResponse object for use in analytics adapters as 'pbsBidId'
+          const bidId = utils.deepAccess(bid, 'ext.prebid.bidid');
+          if (typeof bidId === 'string' && !utils.isEmptyStr(bidId)) {
+            bidObject.pbsBidId = bidId;
+          }
+
+          // store wurl by auctionId and adId so it can be access from the BID_WON event handler
+          if (typeof bid.wurl === 'string' && !utils.isEmptyStr(bid.wurl)) {
+            addWurl(bidRequest.auctionId, bidObject.adId, bid.wurl);
+          }
+
+          let extPrebidTargeting = utils.deepAccess(bid, 'ext.prebid.targeting');
 
           // If ext.prebid.targeting exists, add it as a property value named 'adserverTargeting'
           if (extPrebidTargeting && typeof extPrebidTargeting === 'object') {
+            // If wurl exists, remove hb_winurl and hb_bidid targeting attributes
+            if (bid.wurl) {
```

Comment:
seatbid[].bid[].ext.prebid.event.win not wurl

please add a comment:

// The removal of hb_winurl and hb_bidid targeting values is temporary
// once we get through the transition, this block will be removed.